sylvia/tests/reply.rs

```diff
@@ -198,7 +214,10 @@ where
         &self,
         ctx: ReplyCtx<Q>,
         data: Option<Binary>,
-        _payload: Binary,
+        // Blocked by https://github.com/CosmWasm/cw-multi-test/pull/216.
```

Comment:
Let's add a TODO mark and URL to an issue that will solve this blocked part

Reply:
Good point. We won't release this feature until fix in MultiTest is released.

I will add `TODO` in following PR as there are other changes to be done.